app/models/user.rb

```diff
@@ -70,14 +68,16 @@ class User < ApplicationRecord
   has_many :wellplates, through: :collections
   has_many :screens, through: :collections
   has_many :research_plans, through: :collections
+  has_many :vessels, through: :collections
+  has_many :created_vessels, class_name: 'Vessel', inverse_of: :creator
 
   has_many :samples_created, foreign_key: :created_by, class_name: 'Sample'
 
   has_many :sync_out_collections_users, foreign_key: :shared_by_id, class_name: 'SyncCollectionsUser'
```

Comment:
Rest of file contains changes by auto-linter / rubocop autoformat. 
= lots of syntactical changes with no functional effects. 

If desired I can revert this as rubocop changes are essentially unrelated.

However we will need to address this at some point as this will be performed on every auto-format (which for my part I run very frequently) also it hints to some code smells (e.g. the "foreign_key:" parameter does not exist on `has_many`, only on `belongs_to`).